firebase-perf/src/test/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidatorTest.java

```diff
@@ -189,6 +189,14 @@ public void testInvalidCustomAttribute() {
     trace.clearCustomAttributes().putCustomAttributes("ga_test", "value");
     assertThat(new FirebasePerfTraceValidator(trace.build()).isValidPerfMetric()).isFalse();
 
+    trace = trace.clone();
+    trace.clearCustomAttributes().putCustomAttributes("key", "");
+    assertThat(new FirebasePerfTraceValidator(trace.build()).isValidPerfMetric()).isFalse();
+
+    trace = trace.clone();
+    trace.clearCustomAttributes().putCustomAttributes("value", "");
```

Comment:
Do you want to test `trace.clearCustomAttributes().putCustomAttributes("", "value");`

Reply:
Ha! Good catch!